lib/maths/CTimeSeriesModel.cc

```diff
@@ -112,6 +112,7 @@ double aggregateFeatureProbabilities(const TDouble4Vec& probabilities, double co
 
 const std::string VERSION_6_3_TAG("6.3");
 const std::string VERSION_6_5_TAG("6.5");
+const std::string VERSION_6_8_TAG("6.8");
```

Comment:
I think this should make clear that it starts from 6.8.14 and not 6.8.0.

```suggestion
const std::string VERSION_6_8_14_TAG("6.8.14");
```

Reply:
Done in [6add0d2](https://github.com/elastic/ml-cpp/pull/1674/commits/6add0d230149bdfab359c78a32925eec43da26ed).